test/acceptance/authz/nodes_test.go

```diff
@@ -86,6 +86,10 @@ func TestAuthzNodes(t *testing.T) {
 		helper.AddPermissions(t, adminKey, customRole, helper.NewNodesPermission().WithAction(authorization.ReadNodes).WithVerbosity(verbosity.OutputMinimal).Permission())
 	})
 
+	t.Run("assign custom role to custom user", func(t *testing.T) {
+		helper.AssignRoleToUser(t, adminKey, customRole, customUser)
+	})
```

Comment:
Is this change connected to the role prefixing?

Reply:
that was a probelm in the test and got discovered by the prefix